variables.tf

```diff
@@ -116,6 +116,24 @@ variable "tags" {
   default     = {}
 }
 
+variable "internet_gateway_block_enabled" {
```

Comment:
Please update this variable.

Reply:
@antonbabenko I update both variables - `vpc_block_public_access_options` and `vpc_block_public_access_exclusions`. I hope that you are ok with detailed description for each variable, which later is presented in Terraform docs.